scripts/apply-wmchat-consent-live.ps1

```
@@ -0,0 +1,662 @@
+        [System.IO.Path]::GetFullPath($CommonDir)
+    } else {
+        [System.IO.Path]::GetFullPath((Join-Path $Worktree $CommonDir))
+    }
+    if (-not (Test-Path -LiteralPath $CommonDirFull -PathType Container)) {
+        Fail 34 "Release worktree common Git directory is missing."
+    }
+    & git -C $Worktree fetch --quiet origin $ApprovedRemoteBranch
+    if ($LASTEXITCODE -ne 0) { Fail 34 "Unable to freshly fetch origin/$ApprovedRemoteBranch." }
+    & git -C $Worktree merge-base --is-ancestor $ReleaseCommit "origin/$ApprovedRemoteBranch"
+    if ($LASTEXITCODE -ne 0) { Fail 34 "ReleaseCommit is not contained in freshly fetched approved branch." }
+    $CommittedScript = Invoke-GitText $Worktree @("rev-parse", "$ReleaseCommit`:$ScriptRelativePath") "Release commit does not contain this runner."
```

PSScriptAnalyzer (1.25.0)

[info] Cmdlet 'Invoke-GitText' has positional parameter. Please use named parameters instead of positional parameters when calling a command.

(PSAvoidUsingPositionalParameters)